Upstream-Status: Pending

automake 1.12 has deprecated  automatic de-ANSI-fication support

this patch avoids this error with automake 1.12

| configure.ac:32: error: automatic de-ANSI-fication support has been removed
...
| automake: warnings are treated as errors
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: warning: 'libjpeg.la': linking libtool libraries using a non-POSIX
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'


Signed-Off-By: Nitin A Kamble <nitin.a.kamble at intel.com>
2012/05/02

Index: jpeg-8c/configure.ac
===================================================================
--- jpeg-8c.orig/configure.ac
+++ jpeg-8c/configure.ac
@@ -28,9 +28,6 @@ AM_INIT_AUTOMAKE([-Wall -Werror no-dist
 # with --disable-silent-rules or use "make V=1".
 AM_SILENT_RULES([yes])
 
-# This is required when using the de-ANSI-fication feature.
-AM_C_PROTOTYPES
-
 # Add configure option --enable-maintainer-mode which enables
 # dependency checking and generation useful to package maintainers.
 # This is made an option to avoid confusing end users.
@@ -44,6 +41,10 @@ AC_PROG_INSTALL
 AC_PROG_MAKE_SET
 AC_PROG_LN_S
 
+# automake 1.12 seems to require this, but automake 1.11 doesn't recognize it
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
+
 # Check if LD supports linker scripts,
 # and define automake conditional HAVE_LD_VERSION_SCRIPT if so.
 AC_ARG_ENABLE([ld-version-script],
